v2.0/enable-node-map.md

```diff
@@ -4,4 +4,199 @@ summary: Learn how to enable the node map in the Admin UI.
 toc: false
 ---
 
-*Docs coming soon*
\ No newline at end of file
+The **Node Map** visualizes the geographical configuration of a multi-regional cluster by plotting the node localities on a world map. The **Node Map** also provides real-time cluster metrics, with the ability to drill down to individual nodes to monitor and troubleshoot the cluster health and performance.
+
+This page walks you through the process of setting up and enabling the **Node Map**.
+
+{{site.data.alerts.callout_info}}The <b>Node Map</b> is an <a href="enterprise-licensing.html">enterprise-only</a> feature. However, you can <a href="https://www.cockroachlabs.com/pricing/request-a-license/">request a trial license</a>  to enable the <b>Node Map</b>. {{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Setting Up and Enabling the Node Map
+
+To enable the **Node Map**, you need to start the cluster with the correct `--locality` flags and assign the latitudes and longitudes for each locality.
+
+{{site.data.alerts.callout_info}}The <b>Node Map</b> won't be displayed until <i>all</i> nodes are started with the correct <code>--locality</code> flags and all localities are assigned the corresponding latitudes and longitudes. {{site.data.alerts.end}}
+
+Consider a scenario of a four-node geo-distributed cluster with the following configuration:
+
+|  Node | Region | Datacenter |
+|  ------ | ------ | ------ |
+|  Node1 | us-east-1 | us-east-1a |
+|  Node2 | us-east-1 | us-east-1b |
+|  Node3 | us-west-1 | us-west-1a |
+|  Node4 | eu-west-1 | eu-west-1a |
+
+### Step 1. Ensure the CockroachDB Version is 2.0 or Higher
+
+~~~ shell
+$ cockroach version
+~~~
+
+If not, [upgrade to CockroachDB v2.0](upgrade-cockroach-version.html).
```

Comment:
Maybe first have a sentence with something about what you're supposed to see?

Reply:
Yes, let's put this as a second code block showing the response:

```
~~~
Build Tag:   {{page.release_info.version}}
Build Time:  {{page.release_info.build_time}}
Distribution: CCL
Platform:     darwin amd64
Go Version:   go1.8.3
C Compiler:   4.2.1 Compatible Clang 3.8.0 (tags/RELEASE_380/final)
Build SHA-1:  5b757262d33d814bda1deb2af20161a1f7749df3
Build Type:   release
~~~
```

Reply:
Following that, I'd write out the sentence a bit more explicitly, e.g.:

> If any node is running an earlier version, [upgrade it to CockroachDB v2.0](upgrade-cockroach-version.html).

Reply:
Added and updated to match 2.0 release requirements (Go version, SHA)